tools/modules/EnsEMBL/Web/Component/Tools/VEP/InputForm.pm

```diff
@@ -514,6 +518,35 @@ sub _build_variants_frequency_data {
       })]
     }) if (first { $_->{'value'} eq 'Homo_sapiens' } @$species);
 
+    my @custom_frequencies = @{$self->_get_customs_by_section($current_section)};
+    my $frequncy_species_data = {};
```

Comment:
Misspelled variable, could you please fix this variable in this file?
```suggestion
    my $frequency_species_data = {};
```

Reply:
updated, thanks Nuno!